script/build_hosts

```diff
@@ -1,21 +1,25 @@
 #!/usr/bin/python3
 
 import yaml
+import argparse
 
+parser = argparse.ArgumentParser(description='script to build hosts.yml')
+parser.add_argument('-w', action='store_true', dest='wf', help='enable world finals mode')
+args = parser.parse_args()
+print(vars(args))
 contestants = {}
 judges = {}
 others = {}
-wf = 0
-if wf:
+if args.wf:
     name = "wf2021"
     timezone = "Asia/Dhaka"
     max_team = 152
     max_human_judge = 12
-    max_pc2_autojudge = 0
-    max_kattis_autojudge = 16
+    max_pc2_autojudge = 16
+    max_kattis_autojudge = 0
     max_domjudge = 16
     max_domjudge_ccsadmin = 5
```

Comment:
We will be 7 DJ people in Dhaka - perhaps we can at least make room in the hosts file for this even if we don't end up with 7 machines?

Reply:
sure